### PR TITLE
Fixed ticket_6217

### PR DESCRIPTION
**Pull request by keshav2212**
I am new towards contributions and I have tried to solve this problem.

### CODE REVIEW

**Comment by keshav2212** [comment]
ok..so now should I close this pull request ?

**Comment by stsewd** [comment]
No, you should push your new changes on this branch

**Comment by keshav2212** [comment]
ok, then I need to create new pull request ?

**Comment by stsewd** [comment]
No, this pull request is updated when you push new changes. You should check https://opensource.guide/how-to-contribute/#opening-a-pull-request

**Comment by keshav2212** [comment]
Here, I am again tried to remove 'Search_Analytics' Flag. Can you give review on this ?

File: readthedocs/projects/urls/private.py [hunk 1]
```diff
@@ -129,11 +128,6 @@
         r'^(?P<project_slug>[-\w]+)/advertising/$',
         ProjectAdvertisingUpdate.as_view(), name='projects_advertising',
     ),
-    url(
-        r'^(?P<project_slug>[-\w]+)/search-analytics/$',
-        SearchAnalytics.as_view(),
-        name='projects_search_analytics',
```
Comment (stsewd) [inline readthedocs/projects/urls/private.py]: Only the flag should be removed, not the functionality

**Comment by keshav2212** [comment]
can you review it ?

**Comment by stsewd** [comment]
This was done in https://github.com/readthedocs/readthedocs.org/pull/6447

**Pull request closed without merging**